src/main/resources/story/web/deleteBoard.story

```diff
@@ -0,0 +1,41 @@
+Scenario: Create variables for story
+Given I initialize story variable `boardName` with value `#{generate(regexify '[a-z]{5}[A-Z]{2}')}`
+Given I initialize story variable `nameList1` with value `#{generate(Ancient.god)}`
+Given I initialize story variable `nameCard1` with value `#{generate(Ancient.hero)}`
+Given I initialize story variable `nameList2` with value `#{generate(Ancient.titan)}`
+Given I initialize story variable `nameCard2` with value `#{generate(Animal.genus)}`
+
+Scenario: Log-in
+When I login with ${email} and ${password}
+When I click on element located `id(login-submit)`
+
+Scenario: Create a new board
```

Comment:
Scenario with board and cards creation already exist.
Please either create corresponding composite steps or use previosuly created stories as preconditions
![image](https://user-images.githubusercontent.com/60129090/216253021-f81b28be-2cb7-4cd4-a5b2-630f9a584b23.png)
![image](https://user-images.githubusercontent.com/60129090/216253089-98e6176a-327b-49c3-bf67-64a877935e6d.png)